src/WhitelistNFT.sol

```diff
@@ -31,4 +37,21 @@ contract WhitelistNFT is ERC721, Ownable {
         require(owner != address(0), "ZERO_ADDRESS");
         return super.balanceOf(owner);
     }
+
+
+    //@dev: _update is a function that is called by the internal transfer functions
```

Comment:
I guess, the comment can be updated to reflect, that it is internally called for transfers, burns and mints! Will make it easy to understand the scope at which it applies.

Reply:
agreed